lib/Sema/CSFix.h

```diff
@@ -592,7 +597,32 @@ class DefineMemberBasedOnUse final : public ConstraintFix {
                                         DeclName member,
                                         ConstraintLocator *locator);
 };
-	
+
+class AllowProtocolTypeMember final : public ConstraintFix {
```

Comment:
I think the name has to get adjusted to something like `AllowMemberRefOnExistential` or similar to point out what the problem is.